src/server/worldserver/worldserver.conf.dist

```diff
@@ -2901,7 +2901,7 @@ Arena.RatingDiscardTimer = 600000
 #        Default:     0 - (Disabled)
 #                     1 - (Enabled)
 
-Arena.AutoDistributePoints = 0
+Arena.AutoDistributePoints = 1
```

Comment:
Line 2901 too

Reply:
you mean the default description? gonna do it

btw you can do it by clicking "edit this file" :P